docs/src/development.md

```diff
@@ -47,7 +47,7 @@ julia> using Revise; using Trixi
 ```
 You can run a simulation by executing
 ```julia
-julia> Trixi.run("examples/2d/parameters.toml")
+julia> Trixi.run("examples/2d/parameters_advection_basic.toml")
```

Comment:
I decided to use only minimal changes for the old TOML parameter files. Since these aren't really *elixirs* as we will have them in Taal, I kept the prefix `parameters_` for them. All new versions are called `elixir_*.jl`.

Reply:
Ah yes, I missed the TOML extension. Sorry about that.